cli/engine/ApplyEngine.py

```diff
@@ -39,7 +36,7 @@ def __init__(self, input_data):
         self.infrastructure_docs = []
         self.manifest_docs = []
 
-        self.__ping_retries: int = input_data.ping_retries
+        self.ping_retries: int = input_data.ping_retries
```

Comment:
Is there any reason for this variable to be public? I don't see any direct usage of it.

Reply:
I would agree with you, however the __ paradigm is currently not used  inside the CLI code. So for me this is currently just about consistency. New stuff like the dependency code you are working on or PoC work we should use it and when time allows add it in bulk to the current CLI code as well.